mixins/textInput.js

```diff
@@ -114,29 +114,37 @@ export default {
     })
   },
 
+  iconClass() {
```

Comment:
_technically_ it's not a class, it's just a string referencing the type of icon. sure, it gets turned into a class in the icon helper through 'fa-' + props.icon, but this method isn't __technically__ returning a class at this point... /pedantry